server/container_create_linux.go

```diff
@@ -245,6 +245,10 @@ func (s *Server) createSandboxContainer(ctx context.Context, ctr ctrfactory.Cont
 
 	imageName := imgResult.SomeNameOfThisImage
 	imageID := imgResult.ID
+	imageRef := ""
```

Comment:
Consider renaming this to `someRepoDigest` as well

Reply:
Yep, done :+1: